src/gsi/stprad.f90

```diff
@@ -313,7 +313,7 @@ subroutine stprad(radhead,dval,xval,rpred,spred,out,sges,nstep)
 
         endif
 
-!$omp parallel do schedule(dynamic,1) private(nn,ic,mm,ncr,k,kk,rad,val,val2,cg_rad,wnotgross,wgross)
+! !$omp parallel do schedule(dynamic,1) private(nn,ic,mm,ncr,k,kk,rad,val,val2,cg_rad,wnotgross,wgross)
```

Comment:
We turn off threading here.   Can we fix the threading or should we totally remove it for this section?

Reply:
Russ,

We may be able to fix this threading, so I would prefer to leave it in for a later update.  Not updating now because I think we should get the other changes in ASAP to fix the trunk.

Reply:
Got it.  Commented out line retained for future work.